.github/workflows/release-build-publish.yml

```diff
@@ -32,24 +32,44 @@ jobs:
           VERSION: ${{ github.event.inputs.version }}
 
       - name: Build linux archives
-        if: ${{ runner.os == 'Linux' }}
         run: make packaging
         env:
           VERSION: ${{ github.event.inputs.version }}
 
-      - name: Upload archives as actions artifact
-        uses: actions/upload-artifact@v2
-        with:
-          name: distributions
-          path: build/dist/
-
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v1
         with:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: us-east-1
 
+      - name: Download package signing GPG secret key
+        run: |
+          aws secretsmanager get-secret-value --region us-east-1 --secret-id "XRayDaemonSecrets" | jq -r ".SecretString" > aws-xray-secret.gpg
+          md5sum aws-xray-secret.gpg
+
+      - name: Import signing GPG key
+        run: |
+          gpg --import aws-xray-secret.gpg
+          gpg --list-keys
```

Comment:
The above comment of importing public GPG key apply here as well.

Reply:
Same reason as above.